rotkehlchen/chain/bitcoin/xpub.py

```diff
@@ -199,12 +199,10 @@ def _derive_xpub_addresses(
             known_addresses = getattr(self.db.get_blockchain_accounts(cursor), blockchain.value.lower())  # noqa: E501
 
         new_addresses = []
-        new_balances = []
         existing_address_data = []
         for entry in derived_addresses_data:
             if entry.address not in known_addresses:
                 new_addresses.append(entry.address)
-                new_balances.append(entry.balance)
```

Comment:
I was reading all the code about xpub the other day when reading PRs and saw that this variable was not being used since we read all the balances later